client/my-sites/site-settings/form-jetpack-monitor.jsx

```diff
@@ -1,212 +1,179 @@
+	isUpdatingSiteMonitorSettings
+} from 'state/selectors';
+
+class SiteSettingsFormJetpackMonitor extends Component {
+	state = {};
+
+	componentWillReceiveProps( nextProps ) {
+		if ( isEmpty( this.state ) && nextProps.monitorSettings ) {
+			this.setState( nextProps.monitorSettings );
+		}
+	}
 
-		site.fetchMonitorSettings( ( function( error, data ) {
-			if ( ! error ) {
-				this.setState( {
-					emailNotifications: data.settings.email_notifications,
-					wpNoteNotifications: data.settings.wp_note_notifications
-				} );
-			} else {
-				debug( 'error getting Monitor settings', error );
-			}
-		} ).bind( this ) );
+	recordEvent = ( event ) => {
+		return () => {
+			this.props.trackEvent( event );
+		};
+	}
 
+	handleToggle = name => () => {
+		this.props.trackEvent( `Toggled ${ name }` );
 		this.setState( {
-			enabled: site.isModuleActive( 'monitor' )
-		} );
-	},
+			...this.state,
+			[ name ]: ! this.state[ name ],
+		}, this.saveSettings );
+	}
 
-	prompt: function() {
-		return (
-			<div className="site-settings__jetpack-prompt">
-				<img src="/calypso/images/jetpack/illustration-jetpack-monitor.svg" width="128" height="128" />
-
-				<div className="site-settings__jetpack-prompt-text">
-					<p>{ this.translate( "Automatically monitor your website and make sure it's online." ) }</p>
-					<p>
-						{ this.translate(
-							"We'll periodically check your site from our global network of servers to make sure it's online, and email you if it looks like your site is not responding for any reason."
-						) }
-					</p>
-				</div>
+	saveSettings = () => {
+		const { siteId } = this.props;
 
-			</div>
-		);
-	},
+		this.props.updateSiteMonitorSettings( siteId, this.state );
+	}
 
-	saveSettings: function() {
-		this.setState( { submitingForm: true } );
-		notices.clearNotices( 'notices' );
-		this.props.site.updateMonitorSettings( this.state.emailNotifications, this.state.wpNoteNotifications, ( function( error ) {
-			if ( error ) {
-				this.handleError();
-				notices.error( this.translate( 'There was a problem saving your changes. Please, try again.' ) );
-				return;
-			}
-			notices.success( this.translate( 'Settings saved successfully!' ) );
-			this.props.markSaved();
-			this.setState( { submittingForm: false } );
-		} ).bind( this ) );
-	},
+	settingsMonitorEmailCheckbox() {
+		const { monitorActive, translate } = this.props;
 
-	settings: function() {
 		return (
-			<div>
-				<p>{ this.translate( "Jetpack is currently monitoring your site's uptime." ) }</p>
-				{ this.settingsMonitorEmailCheckbox() }
-				{ config.isEnabled( 'settings/security/monitor/wp-note' ) ? this.settingsMonitorWpNoteCheckbox() : '' }
-			</div>
+			<CompactFormToggle
+				disabled={ this.disableForm() || ! monitorActive }
+				onChange={ this.handleToggle( 'email_notifications' ) }
+				checked={ !! this.state.email_notifications }
+			>
+				{ translate( 'Send notifications to your {{a}}WordPress.com email address{{/a}}', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Send notifications to your {{a}}WordPress.com email address{{/a}}.' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).